test/dynamo/test_unspec.py

```diff
@@ -199,6 +199,19 @@ def fn(x, y):
             res = opt_fn(x, y)
             self.assertTrue(same(ref, res))
 
+    def test_shape_graph_break(self):
+        from torch._dynamo.comptime import comptime
+
+        def fn(x):
+            x_shape = x.size()
+            comptime.graph_break()
```

Comment:
should we use `comptime.graph_break()` everywhere now, instead of `torch._dynamo.graph_break()` ?

Reply:
uhh, it doesn't really matter. If people like, I can prefer `torch._dynamo.graph_break` and delete the comptime version